### PR TITLE
Add more detailed steps for beginner

### PR DESCRIPTION
**Pull request by pxalcantara**
I've just made my first contribution to the project so, I think would be good to add these points that I've struggled a little on the beginning and make the first contribution for other beginners easier

### CODE REVIEW

**Comment by pxalcantara** [comment]
@maryaB-osr How can I fix this CI fail, Could you help me?
Thank you!

**Comment by chapulina** [comment]
> How can I fix this CI fail

No worries about this one, it's an issue with our infrastructure

File: contributing.md [hunk 1]
```diff
@@ -173,6 +173,7 @@ get aquainted with this development process.
 1. **Fork the Ignition library** you want to contribute to. This will create
    your own personal copy of the library. All of your development should
    take place in your fork.
+   - An important thing to do is create an remote pointing to the [upstream remote repository](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork). This way, you can always check some modification on the original repository and **always** keep your fork repository updated.
```
Comment (maryaB-osr) [inline contributing.md:176]:
```suggestion
   - An important thing to do is create a remote pointing to the [upstream remote repository](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork). This way, you can always check for modifications on the original repository and **always** keep your fork repository up to date.
```

File: contributing.md [hunk 2]
```diff
@@ -183,7 +184,10 @@ get aquainted with this development process.
    a release / master branch. This is a good habit to get in, and will make
    your life easier.
 
-1. **Write your code.** This is the fun part.
+1. **Write your code.** This is the fun part, but is good to remeber:
+   - Always [sign your commits](https://docs.github.com/en/github/authenticating-to-github/signing-commits);
+   - Look the existing code and try to keep the pattern used as much as possible;
+   - **Alway** keep your branch updated with the original repository;
```
Comment (maryaB-osr) [inline contributing.md:190]:
```suggestion
1. **Write your code.** This is the fun part, but is good to remember:
   - Always [sign your commits](https://docs.github.com/en/github/authenticating-to-github/signing-commits) (See the bullet about Developer Certificate of Origin in the [Process](#markdown-header-process) section below)
   - Look at the existing code and try to maintain the existing style and pattern as much as possible
   - **Always** keep your branch updated with the original repository
```

**Comment by pxalcantara** [comment]
Thank you @maryaB-osr suggestions accepted!

**Pull request merged**